test/CMakeLists.txt

```diff
@@ -28,10 +28,6 @@ find_package(Catch2 CONFIG REQUIRED)
 
 include(Catch)
 
-add_library(catch_main OBJECT catch_main.cpp)
```

Comment:
Could you elaborate on what the benefits are of keeping the catch_main target? When keeping it it just serves as an alias for Catch2::CatchWithMain. My initial thought was to remove it, since it shortens and simplifies the cmake file and thus servers the purpose of a minimal starter project better.